doc/sections/tactics.md

```diff
@@ -75,6 +75,29 @@ The tactic `apply t` replaces a goal of type `T` by possibly new
 goals `?0` of type `TO`, ..., `?n` of type `Tn` if `t` is of type
 `∀x1:T1,..,∀xn:Tn,?0` and `t ?1 .. ?n` is of type `T`.
 
+<!---------------------------------------------------------------------------->
+### `why3`
+
+The tactic `why3` calls a prover (using the why3 platform) to solve the
+current goal. The user can specify the prover in two ways :
+- by using the command `set prover` (described in [commands](commands.md))
```

Comment:
- globally by using the command ...
- locally by ...

Reply:
Fixed in a29414085332857be96b62ab67a96960b46d746d